pkg/build/admission/defaults/admission.go

```diff
@@ -83,6 +83,15 @@ func (a *buildDefaults) applyBuildDefaults(build *buildapi.Build) {
 		addDefaultEnvVar(envVar, buildEnv)
 	}
 
+	sourceDefaults := a.defaultsConfig.SourceStrategyDefaults
+	sourceStrategy := build.Spec.Strategy.SourceStrategy
+	if sourceDefaults != nil && sourceDefaults.Incremental != nil && *sourceDefaults.Incremental &&
```

Comment:
it already defaults to false...... if the user didn't express a preference when they created the buildconfig, then it's going to run w/ incremental=false.  So there's no point in creating a cluster-wide default that says "if the user didn't express a preference, run with false" because that's what already happens.